readthedocs/projects/models.py

```diff
@@ -253,6 +253,9 @@ def sync_supported_versions(self):
                 verbose_name__in=supported).update(supported=False)
             self.versions.filter(verbose_name=LATEST_VERBOSE_NAME).update(supported=True)
 
+    def user_is_admin(self, user):
```

Comment:
We are trying to unify all this logic in one place, which is in the privacy app. This should be routing that logic through there, at least.

https://github.com/rtfd/readthedocs.org/blob/master/readthedocs/privacy/backend.py#L145-L149